extensions/container-image/runtime/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>quarkus-container-image-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-container-image</artifactId>
+    <name>Quarkus - Container - Runtime</name>
+    <description>Build container images of your application</description>
```

Comment:
Can we add something here saying that we only need this in order to make doc generation happy?

Reply:
Sure. Let me get to it.

```
Build container images of your application. Only needed to make sure that the container image configuration is generated from the io.quarkus.container.image.deployment.ContainerImageConfig. See https://github.com/quarkusio/quarkus/issues/12316. 
``` 

What do you think about that?

Reply:
LGTM